src/include/users/cmdline.rb

```diff
@@ -390,7 +390,7 @@ def UserDeleteHandler(options)
       out = Convert.to_map(
         SCR.Execute(
           path(".target.bash_output"),
-          Builtins.sformat("ps --no-headers -u %1", uid)
+          Builtins.sformat("/usr/bin/ps --no-headers -u %1", uid.to_i)
```

Comment:
Why to_i?

Reply:
If uid cannot be converted to a proper number, the result will be `0` which, after all, it is the root's UID. In this case, as YaST runs as root, the only potential problem is that detecting whether the user is logged in or not could fail (and it will prevent deleting the user).

For me it is fine, but I just wanted to point it out.

Reply:
yes, it is true, but it should not be security issue and also code above handles it ( so it "should" not happen ). This is mainly last defense against malformed input.